src/controllers/action-queue/push.js

```diff
@@ -88,11 +86,12 @@ export async function prepareNewPushEvents(
   return pushEvents
 }
 
-export async function checkPushEvents(account: EdgeAccount, eventIds: string[]): Promise<boolean> {
+export async function checkPushEvents(context: ExecutionContext, eventIds: string[]): Promise<boolean> {
+  const { account, clientId } = context
   const { rootLoginId: loginId } = account
   const requestBody: PushRequestBody = {
     apiKey: AIRBITZ_API_KEY,
-    deviceId,
+    deviceId: clientId,
```

Comment:
Why was PushRequestBody not updated to clientId?

Reply:
`deviceId` is apart of the push-server API spec. Changing this requires coordination with the push-server implementation. Now may be the time to push for such a change, but we can also be lazy here haha